jwst-masterclass/Dockerfile

```diff
@@ -28,11 +28,30 @@ USER root
 RUN apt-get update --yes > /dev/null && \
     apt-get install --yes curl && \
     apt-get install --yes rsync && \
+    apt-get install --yes dbus-x11 && \
```

Comment:
This is faster if you put all of them in one go - so `apt-get install --yes package1 package2' etc